jac-cloud/jac_cloud/tests/test_simple_graph_mini.py

```diff
@@ -0,0 +1,416 @@
+"""JacLang Jaseci Unit Test."""
```

Comment:
Is there anyway we can share the test methods between jac-cloud and jac-mini? e.g., maybe inherit the test class and change the setup logic depending if its mini or cloud?

Reply:
Unit Tests also have different checks as jac-mini is very minimal implementations. It also uses different DB